src/test/utils.tsx

```diff
@@ -348,3 +348,52 @@ export const getDownloadButtons = () => {
     });
   return { gcseButton, aqaButton, ocrButton };
 };
+
+const findDashboardButtons = () => {
```

Comment:
We have (incorrectly, in my opinion) declared a few functions in this file that are only used in a single test file, so I understand why you've added all this new code here, but I think it's wrong. This file should only contain utilities that are useful in many tests. Otherwise, this is going to grow out of control very quickly. Bottomline: I think these functions should be moved to the test file where they're being used, as this place should be just for common code.

Reply:
Moved these into their relevant test files